network/connection.go

```diff
@@ -103,7 +104,7 @@ func NewConnectionFromFirstPacket(pkt packet.Packet) *Connection {
 			IP:       pkt.Info().Src,
 			Protocol: uint8(pkt.Info().Protocol),
 			Port:     pkt.Info().SrcPort,
-		}).Init()
```

Comment:
I don't understand why you removed this, please explain. Was there a change to the `Init()` method I was not aware of? Multiple occurences.

Reply:
That method does nothing now and I actually wanted to completely get rid of it but I just removed this one line :facepalm: 

https://github.com/safing/portmaster/blob/e82ec22f0777c5c0a1c4817cf95b312b19dc6a38/intel/entity.go#L76-L79